npmpackagejsonlint.config.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+	// See https://www.npmjs.com/package/@wordpress/npm-package-json-lint-config
+	extends: '@wordpress/npm-package-json-lint-config',
```

Comment:
Alternatively, just list all those rules here and have one package less to Renovate/update. ;-)

Reply:
🤷‍♂ How much value do we get value from extending? I'm inclined to just list our own rules here, it's pretty trivial to pull in or remove a rule when it makes sense, while keeping the package up to date adds overhead and it can be surprising when rules shift around.

Reply:
>How much value do we get value from extending?

Whenever anything changes/gets deprecated in the core, we'd inherit those changes. We have that problem with our Stylelint config - a bunch of it was outdated last time I checked.